care/utils/assetintegration/onvif.py

```diff
@@ -6,7 +6,7 @@
 
 
 class OnvifAsset(BaseAssetIntegration):
-    _name = "onvif"
```

Comment:
Can you confirm if these needs a data migration, i belive these are stored in the database and changing them would need a data migration as well.

Reply:
Hey, after reviewing this, these changes were not required and since `asset_class` is a CharField, using a `filters.CharFilter` works without having to make any additional changes. This should be resolved.